app/views/unauthorised.scala.html

```diff
@@ -17,44 +17,59 @@
 @import config.ExternalServicesConfig
 @import views.helpers.Title
 @import views.html.components.gds._
+@import config.featureFlags.TdrUnauthorisedMsgConfig
 
 @this(
     govukLayout: gdsMainTemplate,
     pageTitle: pageTitle,
     paragraphBody: paragraphBody,
     link: link,
     bulletList: bulletList,
-    config: ExternalServicesConfig
+    config: ExternalServicesConfig,
+    tdrUnauthorisedMsgConfig: TdrUnauthorisedMsgConfig
 )
 
 @()(implicit request: Request[_], messages: Messages)
 
-@howToSection = {
-    @paragraphBody(messages("unauthorised.paragraph.1"))
-
-    @bulletList(List(
-      Html(messages("unauthorised.paragraph.1.bullet.1", link(
-        id = Some("get_eori_link"),
-        text = messages("unauthorised.paragraph.1.bullet.1.link"),
-        call = Call("GET", config.eoriService)
-      ))),
-      link(
-        id = Some("access_cds_link"),
-        text = messages("unauthorised.paragraph.1.bullet.2.link"),
-        call = Call("GET", config.cdsRegister)
-      )
-    ))
-}
+@if(tdrUnauthorisedMsgConfig.isTdrUnauthorisedMessageEnabled) {
```

Comment:
More than the two views, I would like to propose my comment below :smiley:

Reply:
Don't see any gain in splitting it into two views tbh, plus this way confines the logic to a single file rather than also polluting the controller unnecessarily :slightly_smiling_face:

Reply:
I second that, Darryl. Unfortunately our views have been built with a different philosophy.

Reply:
I agree that too much logic in views is to be avoided (especially as the twirl syntax is so odd) but a simple `if` statement won't tax the readers brains too much I feel ;)